test/timezone/test_mathn_compatibility.rb

```diff
@@ -2,7 +2,13 @@
 
 require 'timezone'
 require 'minitest/autorun'
-require 'mathn'
+
+begin
+  require 'mathn'
+rescue LoadError => e
+  raise e until e.path == 'mathn'
+  # Ruby 2.5 doesn't have `mathn` in Stdlib
+end
```

Comment:
It probably makes sense for this entire test to not be run if `mathn` is not in stdlib.

Reply:
> It probably makes sense for this entire test to not be run if mathn is not in stdlib.

It can be loaded manually from external gem, I guess. Perhaps we should add `development_dependency`?

Reply:
I don't think it is necessary to add a development dependency. If `mathn` is not part of the standard library I am fine not testing compatibility going forward. While there is still support for versions of Ruby in which this library is included, I think it makes sense to retain this test - but the entire test should be skipped if that is not the case.

Reply:
OK, I think I got it. Done.